addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -532,6 +532,11 @@ export class WebsitePreview extends Component {
             return;
         }
         ev.preventDefault();
+        // In the middle of a refresh or a redirection, the iframe isn't loaded
+        // yet: do nothing.
+        if (this.websiteService.contentWindow === undefined) {
+            return;
+        }
```

Comment:
Shouldn't this be added before `ev.preventDefault()`? Otherwise you prevent reloading with ctrl-r/f5 as long as the iframe hasn't loaded something.